js/formidable_dashboard.js

```
@@ -1 +1,836 @@
+function _toPrimitive(t, r) { if ("object" != _typeof(t) || !t) return t; var e = t[Symbol.toPrimitive]; if (void 0 !== e) { var i = e.call(t, r || "default"); if ("object" != _typeof(i)) return i; throw new TypeError("@@toPrimitive must return a primitive value."); } return ("string" === r ? String : Number)(t); }
+var frmCounter = /*#__PURE__*/function () {
+  /**
+   * Init frmCounter
+   *
+   * @param {Element} element
+   * @param {object} options
+   * @param {integer} options.timetoFinish - Max time in mileseconds for counter to complete the animation.
+   */
+  function frmCounter(element, options) {
+    _classCallCheck(this, frmCounter);
+    if (!element instanceof Element || !element.dataset.counter) {
```

Biome (2.1.2)

[error] The negation operator is used unsafely on the left side of this binary expression.

Unsafe fix: Wrap the expression with a parenthesis


(lint/suspicious/noUnsafeNegation)